source/faucet/main.d

```diff
@@ -382,6 +382,15 @@ public class Faucet : FaucetAPI
         auto owned_utxo_rng = this.state.owned_utxos.byKeyValue()
             // do not pick already used UTXOs
             .filter!(pair => pair.key !in this.used_utxos);
+
+        auto owned_utxo_len = owned_utxo_rng.take(2).array.length;
```

Comment:
You can use count directly and avoid the `array` call as `length` (or is it `count`) is also terminating the lazy range access.

Reply:
Updated to use `count`.